public/js/monitor.js

```diff
@@ -369,4 +369,24 @@ function addBentoObserver(){
 
   const dropDownMenu = document.querySelector(".mobile-nav.show-mobile");
   dropDownMenu.addEventListener("click", () => toggleDropDownMenu(dropDownMenu));
+
+  if (document.body.dataset.experiment) {
+    const submitBtn = document.querySelector("#scan-user-email input[type='submit']");
+    const createFxaCheckbox = document.getElementById("createFxaCheckbox");
+
+    submitBtn.addEventListener("click", (e)=> {
+      if (createFxaCheckbox.checked) {
+        e.preventDefault();
+        if (typeof(ga) !== "undefined") {
+          // Send "Click" ping for #see-additional-recs click
```

Comment:
Why are we commenting this part out?

Reply:
WIP Comment. Will be fixed during first to-do item above.

Reply:
Revised!